conf/steps/imputation_stitch.config

```diff
@@ -11,6 +11,19 @@
 */
 
 process {
+
+    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:GAWK_NOCOMMA' {
+        ext.args2  = "'BEGIN { OFS=\"\\t\" } { split(\$3, a, \"[,]\"); print \$1, \$2, a[1], a[2] }'"
+        ext.prefix = { "${meta.id}.${meta.chr}.posfile_nocomma" }
+        ext.suffix = "txt"
+        publishDir = [enabled: false]
+    }
+
+    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:BGZIP_NOCOMMA' {
+        ext.prefix = { "${meta.id}.${meta.chr}.posfile_nocomma" }
```

Comment:
I think it is better to remove the `_nocomma` since we now have a single posfile option.

Reply:
Same